support/troubleshooting/troubleshooting-operating-system-issues.adoc

```diff
@@ -10,3 +10,42 @@ toc::[]
 
 // Investigating kernel crashes
 include::modules/investigating-kernel-crashes.adoc[leveloffset=+1]
+
+include::modules/troubleshooting-enabling-kdump.adoc[leveloffset=+2]
+
+include::modules/troubleshooting-enabling-kdump-day-one.adoc[leveloffset=+2]
+
+=== Testing the kdump configuration
```

Comment:
Needs an ID: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings